README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
-    <a href="https://docs.rs/unbug">
-        <img src="assets/unbug.svg" width="200" alt="Unbug logo"/>
+    <a target="_blank" href="https://docs.rs/unbug">
+        <img src="https://raw.githubusercontent.com/greymattergames/unbug/main/assets/unbug.svg" width="200" alt="Unbug logo"/>
```

Comment:
Change URL so crates.io will pull the logo from Github when we publish the next version.